packages/astro-embed-baseline-status/BaselineStatus.astro

```diff
@@ -0,0 +1,293 @@
+			devices and browser versions. This feature might not work in older
+			devices or browsers.`;
+	} else if (baseline === 'widely' && date) {
+		return `This feature is well established and works across many
+			devices and browser versions. It’s been available across browsers
+			since ${date}`;
+	}
+	return BASELINE_DEFS[baseline].defaultDescription;
+}
+
+const featureData = await safeGet(API_ENDPOINT + Astro.props.id);
+const feature: Feature = featureData?.baseline
+	? (featureData as Feature)
+	: {
+			baseline: {
+				status: 'no_data',
+			},
+			name: Astro.props.id || 'Unknown feature',
+	  };
+
+const baseline = feature.baseline.status || 'no_data';
+
+const title = BASELINE_DEFS[baseline].title;
+const baselineDate = getBaselineDate(feature);
+const description = getDescriptionDate(baseline, baselineDate);
+const year =
+	baseline === 'newly' && baselineDate ? baselineDate.split(' ')[1]! : '';
+const { chrome, edge, firefox, safari } = feature.browser_implementations || {};
+
+function getAriaLabel(
+	title: string,
+	year: string,
+	newlyAvailable: boolean,
+	chrome = 'no',
+	edge = 'no',
+	firefox = 'no',
+	safari = 'no'
+) {
+	if (title === 'Unknown availability') {
+		chrome = edge = firefox = safari = 'unknown';
+	}
+	return `Baseline: ${title}${year ? ` ${year}` : ''}${
+		newlyAvailable ? ` (newly available)` : ''
+	}. Supported in Chrome: ${
+		chrome === 'available' ? 'yes' : chrome
+	}. Supported in Edge: ${
+		edge === 'available' ? 'yes' : edge
+	}. Supported in Firefox: ${
+		firefox === 'available' ? 'yes' : firefox
+	}. Supported in Safari: ${safari === 'available' ? 'yes' : safari}.`;
+}
+---
+
+<baseline-status>
+	<h1>{feature.name}</h1>
+	<details>
+		<summary
+			aria-label={getAriaLabel(
+				title,
+				year,
+				baseline === 'newly',
+				chrome?.status,
+				edge?.status,
+				firefox?.status,
+				safari?.status
+			)}
+		>
+			<BaselineIcon support={baseline} />
+			<div class="baseline-status-title" aria-hidden="true">
+				<div>
+					{
+						!(baseline === 'limited' || baseline === 'no_data') && (
+							<strong>Baseline</strong>
+						)
+					}
+					{title}
+					{year}
+					{
+						baseline === 'newly' && (
+							<span class="baseline-badge">newly available</span>
+						)
+					}
+				</div>
+				<div class="baseline-status-browsers">
+					<span>
+						<BrowserIcon browser="chrome" />
+						<SupportIcon {baseline} browserImplementation={chrome} />
+					</span>
+					<span>
+						<BrowserIcon browser="edge" />
+						<SupportIcon {baseline} browserImplementation={edge} />
+					</span>
+					<span>
+						<BrowserIcon browser="firefox" />
+						<SupportIcon {baseline} browserImplementation={firefox} />
+					</span>
+					<span>
+						<BrowserIcon browser="safari" />
+						<SupportIcon {baseline} browserImplementation={safari} />
+					</span>
+				</div>
+			</div>
+			<span class="open-icon" aria-hidden="true">
+				<svg
+					xmlns="http://www.w3.org/2000/svg"
```

Comment:
I think the `xmlns` attribute is not needed when the SVGs are inlined in the markup.

Reply:
Removed + optimized the path too.